content/html/guides/Preloading_content/Preloading_content.md

```diff
@@ -0,0 +1,200 @@
+
+This makes it much more likely that the font will be available for the page render, cutting down on FOUT (flash of unstyled text).
+
+This doesn't have to be limited to images, or even files of the same type — think big! You could perhaps preload and display a simple SVG diagram if the user is on a narrow screen where bandwidth and CPU is potentially more limited, or preload a complex chunk of JavaScript then use it to render an interactive 3D model if the user's resources are more plentiful.
+
+## Scripting and preloads
+
+Another nice thing about these preloads is that you can execute them with script. For example, here we create a [`HTMLLinkElement`](/en-US/docs/Web/API/HTMLLinkElement) instance, then attach it to the DOM:
+
+```js
+    var preloadLink = document.createElement("link");
+    preloadLink.href = "myscript.js";
+    preloadLink.rel = "preload";
+    preloadLink.as = "script";
+    document.head.appendChild(preloadLink);
+```
+
+This means that the browser will preload the `myscript.js` file, but not actually use it yet. To use it, you could do this:
+
+```js
+    var preloadedScript = document.createElement("script");
+    preloadedScript.src = "myscript.js";
+    document.body.appendChild(preloadedScript);
+```
+
+This is useful when you want to preload a script, but then defer execution until exactly when you need it.
+
+## Other resource preloading mechanisms
+
+Other preloading features exist, but none are quite as fit for purpose as `<link rel="preload">`:
+
+- `<link rel="prefetch">` has been supported in browsers for a long time, but it is intended for fetching resources that will be used in the **_next_** navigation/page load (e.g. when you go to the next page). This is fine, but isn't useful for the current page! In addition, browsers will give `prefetch` resources a lower priority than `preload` ones — the current page is more important than the next. See [Link prefetching FAQ](/en-US/docs/Web/HTTP/Link_prefetching_FAQ) for more details.
+- `<link rel="prerender">` renders a specified webpage in the background, speeding up its load if the user navigates to it. Because of the potential to waste users bandwidth, Chrome treats `prerender` as a [NoState prefetch](https://developers.google.com/web/updates/2018/07/nostate-prefetch) instead.
+- `<link rel="subresource">` was supported in Chrome a while ago, and was intended to tackle the same issue as `preload`, but it had a problem: there was no way to work out a priority for the items (`as` didn't exist back then), so they all got fetched with fairly low priority.
+- There are a number of script-based resource loaders out there, but they don't have any power over the browser's fetch prioritization queue, and are subject to much the same performance problems.
+
+{{{{{embed-compat:html.elements.link.rel.preload}}}}}
```

Comment:
To me, this now renders without a "Browser compatibility" headline. Is that supposed to be added in rendering somehow? I think it would great to have a headline in the markdown, so the author has the control over this. Or did you intentionally leave out the heading here? On the current MDN page, it is there.

Reply:
The heading should be in the Markdown (its omission here is an error in the Markdown). Not in rendering. Because the renderer can't know, for guide pages, what the heading should be (see e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes#Browser_compatibility, where the heading is an H3 with a different text content)